wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisAccelerations.java

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.kinematics;
+
+import static edu.wpi.first.units.Units.MetersPerSecondPerSecond;
+import static edu.wpi.first.units.Units.RadiansPerSecondPerSecond;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.interpolation.Interpolatable;
+import edu.wpi.first.math.kinematics.proto.ChassisAccelerationsProto;
+import edu.wpi.first.math.kinematics.struct.ChassisAccelerationsStruct;
+import edu.wpi.first.units.measure.AngularAcceleration;
+import edu.wpi.first.units.measure.LinearAcceleration;
+import edu.wpi.first.util.protobuf.ProtobufSerializable;
+import edu.wpi.first.util.struct.StructSerializable;
+import java.util.Objects;
+
+/**
+ * Represents the acceleration of a robot chassis. Although this class contains similar members
```

Comment:
Most of this comment is unnecessary since it's pretty obvious that accelerations are not speeds. The original comment in ChassisSpeeds specifically called out Twist2d due to demonstrated confusion between `ChassisSpeeds.vx` being a velocity and `Twist2d.dx` being a position delta.